proxies/python/handlers/client.py

```diff
@@ -0,0 +1,38 @@
+from devcycle_python_sdk import Configuration, DVCClient, DVCOptions
+
+def handle_client(body, data_store):
+    client_id, sdk_key, options = [body.get(k, None) for k in ('clientId', 'sdkKey', 'options')]
+
+    options = options if options else {}
+
+    if (client_id == None):
+        error = {
+            "errorMessage": "Invalid request: missing clientId"
+        }
+
+        return error, 400
+
+    configuration = Configuration()
+    configuration.api_key['Authorization'] = sdk_key
+
+    # TODO remove this when SDK properly supports passing in this property
```

Comment:
did some cheating here to fix some of the incompatibilities at the proxy level just so I could get the tests to run at all